lisa/wlgen/rta.py

```diff
@@ -726,10 +727,15 @@ class Phase(Loggable):
                           when reaching the end of this phase. Currently only
                           supported when duty_cycle_pct=100
     :type barrier_after: str
+
+    :param nodes_membind: the list of NUMA Nodes.
```

Comment:
Given the variety of origins of parameters to wlgen.rta constructors, it might be a good idea to prefix them so that someone not familiar with NUMA knows it's about NUMA (node being a very generic term). It would give something like `numa_nodes_membind`.

@valschneider Any thoughts on that ?

Reply:
In ~l33t~kernelspeak, node very much means NUMA node. I don't have any big preference towards having (or not having) the "numa" prefix.

Reply:
The thing is that it's not the kernel, it's LISA, and there are already other kind of nodes like this one:
https://github.com/ARM-software/lisa/blob/master/lisa/energy_model.py#L147

I'd rather have something relatively obvious than some undecipherable set of abbreviations. Python never had this sort of restriction, so there is no point in obfuscation :-)
http://port70.net/~nsz/c/c11/n1570.html#6.4.2.1p5

Reply:
I will rename it to numa_nodes_membind.
I tried to make it consistent with rt-app name.